dev/packages/alpha/nginx/0.0.3/dataset/ingress_controller/manifest.yml

```diff
@@ -0,0 +1,23 @@
+title: Nginx ingress_controller logs
+release: ga
+type: logs
+streams:
+- input: logs
+  vars:
+  - name: paths
+    type: text
+    title: Paths
+    multi: true
+    required: true
+    show_user: true
+    default:
+    - /var/log/nginx/access.log*
```

Comment:
Ingress has the same path as the access log?

Reply:
That's wrong of course, will change to "ingress.log".

Reply:
Fixed